app/packages/embeddings/src/tracesToData.tsx

```diff
@@ -39,7 +42,7 @@ export function tracesToData(
           autocolorscale: !isContinuous, // isCategorical || isUncolored,
           colorscale: mappedColorscale,
           color: isCategorical
-            ? color.toCSSRGBString()
+            ? color?.toCSSRGBString()
```

Comment:
Is possibly of color being undefined expected/intended? Should there be fallback?

Reply:
I will add a fallback just in case.